docs/guide/docs/channels/messenger.md

```diff
@@ -5,82 +5,139 @@ title: Facebook Messenger
 
 ## Requirements
 
-Messenger requires you to have a Facebook App and a Facebook Page to setup your bot.
+Messenger requires you to have a Facebook App and a Facebook Page to set up your chatbot connection to their platform.
 
-- [Enable Channel-Messenger Module](../main/module#enabling-or-disabling-modules)
-- [Create a Facebook App](https://developers.facebook.com/docs/apps/)
-- [Create a Facebook Page](https://www.facebook.com/pages/creation/)
-- An HTTPS Endpoint to your bot
-  - Create an HTTPS tunnel to your machine using Ngrok. [**Tutorial**](https://api.slack.com/tutorials/tunneling-with-ngrok)
-  - Using Nginx and Let's Encrypt. [**Tutorial**](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-16-04)
+### Messenger Module
+Navigate to the `\modules` folder and check if the archive `channel-messenger.tgz` exists. 
```

Comment:
The user does not need to verify the `channel-messenger.tgz` archive exists, since it's included by default in Botpress. Installing is simply a matter of enabling the module